docs/services/gcp-services-guide.md

```diff
@@ -348,13 +257,13 @@ You can observe resource creation progression using the following:
 
```

Comment:
```
NAME          STATUS   CLASS-KIND         CLASS-NAME       RESOURCE-KIND   RESOURCE-NAME               AGE
```

All claims have this style output these days, so the above example is not entirely accurate.